pkg/validate/fileobjects/raw.go

```diff
@@ -78,9 +80,22 @@ func (r *Raw) Scoped() (*Scoped, status.MultiError) {
 		DynamicNSSelectorEnabled: r.DynamicNSSelectorEnabled,
 		NSControllerState:        r.NSControllerState,
 	}
+	// Parse the patterns from the flag once.
+	// We ignore the error from ParsePatterns for now, assuming valid input or empty.
+	skipGVKErrors, _ := gvkutil.ParsePatterns(flags.NoAPIServerCheckForGroup)
```

Comment:
This package library shouldn't directly access flags, the flag should be passed in as an arg/struct value. It should follow a similar pattern as [SkipAPIServer](https://github.com/GoogleContainerTools/config-sync/blob/main/cmd/nomos/vet/vet_impl.go#L99). This will help with separation of concerns and testability